src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Hello</title>
+</head>
```

Comment:
Will be great to add other tags, but we discuss it tomorrow

Reply:
Ok, thank you for review, i forgot about <meta>